doc/man3/OCSP_sendreq_new.pod

```diff
@@ -25,6 +26,9 @@ OCSP_REQ_CTX_set1_req, OCSP_sendreq_bio - OCSP responder query functions
 
  OCSP_RESPONSE *OCSP_sendreq_bio(BIO *io, const char *path, OCSP_REQUEST *req);
 
+ int OCSP_REQ_CTX_i2d(OCSP_REQ_CTX *rctx, const char *content_type,
+                      const ASN1_ITEM *it, ASN1_VALUE *req);
+
 Deprecated since OpenSSL 3.0, can be hidden entirely by defining
 B<OPENSSL_API_COMPAT> with a suitable version value, see
 L<openssl_user_macros(7)>:
```

Comment:
Please move the deprecation statement on the `OCSP_REQ_CTX_set1_req()` function into the DESCRIPTION section just above the new hint how to replace its use.

Reply:
No, that's not the way it's done; see ```doc/man3/ERR_error_string.pod``` for example.

Reply:
I see. I had seen examples of putting the deprecation info in the DESCRIPTION section, but having it in SYNOPSIS seems more common.